lib/config.js

```diff
@@ -32,7 +32,7 @@ try {
 } catch {}
 
 class Pattern {
-  constructor (pattern, served, included, watched, nocache, type, isBinary) {
+  constructor (pattern, served, included, watched, nocache, type, isBinary, integrity) {
```

Comment:
Can you also document this new property [here](https://github.com/karma-runner/karma/blob/master/docs/config/02-files.md)?

Reply:
I'll try it.